doc/languages-frameworks/javascript.section.md

```diff
@@ -0,0 +1,113 @@
+# Javascript
```

Comment:
Would be nice to add anchors to other headings as well.

Reply:
Added, let me know if you think it hasn't been properly addressed.